packages/datepicker/src/ux-datepicker.ts

```diff
@@ -134,8 +159,6 @@ export class UxDatepicker implements UxComponent {
     if (this.type.toLowerCase() === 'time') {
       this.textboxValue = moment(newValue).format(this.formatters.time);
     }
-
-    this.showDialog = false;
```

Comment:
@bigopon this is the `valueChanged()` where I removed the closing of the dialog that used to cause problems